driver/normalizer/annotation.go

```diff
@@ -119,9 +81,9 @@ var AnnotationRules = On(Any).Self(
 		On(HasInternalType(pyast.Div)).Roles(OpDivide),
 		On(HasInternalType(pyast.Mod)).Roles(OpMod),
 		// TODO: currently without mapping in the UAST
-		//On(HasInternalType(pyast.FloorDiv)).Roles(OpDivide),
-		//On(HasInternalType(pyast.Pow)).Roles(???),
-		//On(HasInternalType(pyast.MatMult)).Roles(???),
+		On(HasInternalType(pyast.FloorDiv)).Roles(OpDivide, Incomplete),
+		On(HasInternalType(pyast.Pow)).Roles(OpMultiply, Incomplete),
```

Comment:
Yes, but you could argue that a node with only the "Incomplete" role is the same as `Unannotated`. 

On the other hand there is certainly a meaning of "the programmer didn't forget about this node, it knows this node exists but couldn't find any appropriate roles" on a node just tagged with `Incomplete` while an `Unannotated` node could be a slip on the annotation process. Which will help me a lot with the second part of this task since I left the hardest nodes to the end :) so I'll do as you say.

I'll also add a PR with a note about this in BIP-002.